package.json

```diff
@@ -23,10 +23,12 @@
     "serve-static": "^1.14.1",
     "vue": "^3.0.0",
     "vue-router": "^4.0.0-0",
+    "vuelidate": "^0.7.6",
     "vuex": "^4.0.0-0"
   },
   "devDependencies": {
     "@types/jest": "^24.0.19",
+    "@types/vuelidate": "^0.7.15",
```

Comment:
Why do we need this dependencies if we have similar in 16 and 17 lines

Reply:
fixed